lib/reporting/reports/orders_and_fulfillment/order_cycle_customer_totals.rb

```diff
@@ -97,7 +97,7 @@ def line_item_includes
 
         def query_result
           report_line_items.list(line_item_includes).group_by { |e|
-            [e.variant_id, e.price, e.order_id]
+            [e.variant_id, e.price, e.order_id, e.order.distributor_id]
```

Comment:
I think that we don't need this because we are grouping by order_id already. One order has only one distributor.

Reply:
Yep, I think you're right... Good point. 
Will update the PR by removing this.